academy-of-management-review.csl

```diff
@@ -234,16 +252,23 @@
           <group>
             <text macro="container-contributors"/>
             <text macro="secondary-contributors"/>
-            <text variable="container-title" font-style="italic" font-weight="bold"/>
+            <choose>
+              <if type="article-journal article-magazine article-newspaper paper-conference" match="any">
```

Comment:
are you basing the title case for conference proceedings on more than the one example in the guide for authors? That seems like a very odd rule to me.